homeassistant/components/binary_sensor/doorbird.py

```diff
@@ -0,0 +1,65 @@
+"""Support for reading binary states from a DoorBird video doorbell."""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.components.doorbird import DOMAIN
```

Comment:
You're not allowed to overwrite the `DOMAIN` of a platform. That should always be the base component domain, eg `'binary_sensor'` in this case. You can import the doorbird domain as another name:
```py
from homeassistant.components.doorbird import DOMAIN as DOORBIRD_DOMAIN
```